frontend/tests/utils/wmgUtils.ts

```diff
@@ -257,38 +261,38 @@ export const selectTissueAndGeneOption = async (page: Page) => {
 };
 
 export const checkSourceData = async (page: Page) => {
-  //click on source data icon
-  await page.locator('[data-testid="source-data-button"]').click();
+  const sourceDataButton = page.getByTestId(SOURCE_DATA_BUTTON_ID);
+  const sourceDataList = page.getByTestId(SOURCE_DATA_LIST_ID);
+
+  // click on source data icon
+  await sourceDataButton.click();
 
   // number of element displayed on source data
-  const n = await page.locator('[data-testid="source-data-list"] a').count();
+  const n = await sourceDataList.locator("a").count();
 
   // close the pop-up
   /**
-   * (thuang): Sometimes pressing escape once wasn't closing the side panel, so
+   * (thuang): Sometimes closing the panel just once doesn't work, so
    * wrapping this to retry and assert the panel is indeed closed
    */
   await tryUntil(
     async () => {
-      await page.keyboard.press("Escape");
-
```

Comment:
update test steps here, since the new version of Blueprint doesn't work with Escape reliably